src/main/xerus/monstercat/Settings.kt

```diff
@@ -23,6 +24,8 @@ object Settings : SettingsNode("xerus/monsterutilities") {
 	val PLAYERSCROLLSENSITIVITY = create("playerSeekbarScrollSensitivity", 6.0)
 	val PLAYERSEEKBARHEIGHT = create("playerSeekbarHeight", 8.0)
 	val ENABLEEQUALIZER = create("equalizerEnabled", false)
+	val EXPORTCURRENTTITLE = create("exportPlayerTitle", false)
+	val EXPORTDIR = create("playerExportDir", Paths.get("player_current_title.txt"))
```

Comment:
Why `Dir`? Looks like a file to me.

Reply:
Yeah at the beginning I was thinking about a dir where an hardcoded-name text file would be created.

Reply:
a899097197c5150bd197d6f508cd6460878d6a05